src/activities/components/ParticipantTypesEdit.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <div>
+    <div class="row">
+      <QToggle
+        v-if="isFeatureEnabled || isUsingAdvanced"
+        v-model="advancedMode"
+        label="Use participant types"
+        class="q-mt-xs"
+      />
+      <QSpace />
+      <QBtn
+        v-if="participantTypesChanged"
+        label="Revert to series values"
+        color="warning"
+        flat
+        icon="undo"
+        @click="revertParticipantTypes"
+      />
+    </div>
+
+    <template v-if="simple">
+      <QInput
+        :model-value="simple.participantType.maxParticipants"
+        type="number"
+        stack-label
+        outlined
+        class="q-my-md"
+        :label="$t('CREATEACTIVITY.MAX_PARTICIPANTS')"
+        :hint="$t('CREATEACTIVITY.MAX_PARTICIPANTS_HELPER')"
+        :placeholder="$t('CREATEACTIVITY.UNLIMITED')"
+        :error="hasError('maxParticipants')"
+        :error-message="firstError('maxParticipants')"
+        input-style="max-width: 100px"
+        :bg-color="simple.changed.maxParticipants ? 'orange-1' : null"
+        @update:model-value="val => (simple.participantType.maxParticipants = val ? parseInt(val) : null)"
+      >
+        <template #before>
+          <QIcon name="group" />
+        </template>
+        <QSlider
+          v-if="simple.participantType.maxParticipants > 0 && simple.participantType.maxParticipants <= 10"
+          v-model="simple.participantType.maxParticipants"
+          :min="1"
+          :max="10"
+          label
+          markers
+          class="q-mx-sm self-end"
+          style="min-width: 60px"
+        />
+        <template #after>
+          <QBtn
+            v-if="simple.changed.maxParticipants"
+            icon="undo"
+            unelevated
+            dense
+            @click="simple.participantType.maxParticipants = simple.seriesParticipantType.maxParticipants"
+          />
+        </template>
+      </QInput>
+    </template>
+    <template v-else>
+      <QCard
+        v-for="({ participantType, seriesParticipantType, changed, users }, idx) in entries"
+        :key="participantType.id || `new-${idx}`"
+        flat
+        bordered
+        class="participant-type"
+      >
+        <QCardSection>
+          <div class="text-h6">
+            Participant type {{ idx + 1 }}
```

Comment:
Not sure if this is actually useful to display.. could be nicer without the title?

Reply:
I agree, I removed it...